### PR TITLE
Content updates from support

### PR DESCRIPTION
**Pull request by xavierlepretre**
This PR adjusts the content following notifications from IDA participants.

### Change scope

The changes in this Pull Request include (please tick all that apply):

* [ ] Small language/grammar fixes
* [x] Small content fixes 
* [ ] Addition of new content
* [x] Sample code/command updates
* [ ] Platform fixes
* [ ] Other

### CODE REVIEW

File: academy/3-my-own-chain/node-api-and-cli.md [hunk 1]
```diff
@@ -243,25 +243,57 @@ It helps to understand the concepts clearly when working hands-on with the Cosmo
...
 $ ./simd keys list
 ```
 
-As you might have expected, you do not have any keys yet:
+</CodeGroupItem>
 
+<CodeGroupItem title="WSL2">
+
+```sh
+$ ./simd keys list --keyring-backend test
```
Comment (xavierlepretre) [inline academy/3-my-own-chain/node-api-and-cli.md:259]: @julienrbrt It is possible we need to do this to cater to those who do it on WSL2.

Comment (julienrbrt) [reply academy/3-my-own-chain/node-api-and-cli.md:259]: What's WSL2?

Comment (xavierlepretre) [reply academy/3-my-own-chain/node-api-and-cli.md:259]: the Windows Subsystem for Linux
https://learn.microsoft.com/en-us/windows/wsl/about

File: academy/2-main-concepts/migrations.md [hunk 2]
```diff
@@ -306,7 +306,7 @@ type App struct {
 }
 ```
 
-And that it is populated:
+Ensure that the `Configurator` is populated:
```
Comment (xavierlepretre) [inline academy/2-main-concepts/migrations.md:309]:
```suggestion
Ensure that the `configurator` is populated:
```

File: tutorials/3-run-node/index.md [hunk 3]
```diff
@@ -314,8 +314,10 @@ Despite this initial balance, before you run your blockchain you still need to e
 
 <HighlightBox type="note">
 
-In this scenario you must meet the 2/3 threshold for validation, so you must stake at least `70000000stake` of your `100000000stake` in the `b9lab` account you just created. Make sure to not use all your stake, so you still have tokens to pay for gas.
-<br></br>
+In this scenario, for your network to even run, you must meet the 2/3 threshold of the weighted validators.
+<br/><br/>
+However you will be alone on the network, so you can stake any number. However, to remind yourself that it is important honest nodes stake a large amount, you stake `70000000stake` of your `100000000stake` in the `b9lab` account you just created. Make sure to not use all your stake, so you still have tokens to pay for gas.
```
Comment (rob-Hitchens) [inline tutorials/3-run-node/index.md:319]: However you will be alone on the network, so you can stake any number. However, to remind yourself that it is important that honest nodes stake a large amount, you stake `70000000stake` of your `100000000stake` in the `b9lab` account you just created. Make sure to not use all of your tokens so you can still pay for gas and so you don't run out of tokens later.

**Review by rob-Hitchens — COMMENTED** [review]
Minor edit

File: tutorials/3-run-node/index.md [hunk 4]
```diff
@@ -314,8 +314,10 @@ Despite this initial balance, before you run your blockchain you still need to e
 
 <HighlightBox type="note">
 
-In this scenario you must meet the 2/3 threshold for validation, so you must stake at least `70000000stake` of your `100000000stake` in the `b9lab` account you just created. Make sure to not use all your stake, so you still have tokens to pay for gas.
-<br></br>
+In this scenario, for your network to even run, you must meet the 2/3 threshold of the weighted validators.
+<br/><br/>
+However you will be alone on the network, so you can stake any number. However, to remind yourself that it is important that honest nodes stake a large amount, you stake `70000000stake` of your `100000000stake` in the `b9lab` account you just created. Make sure to not use all of your tokens so you can still pay for gas and so you don't run out of tokens later.
```
Comment (xavierlepretre) [inline tutorials/3-run-node/index.md:319]: @julienrbrt Does this new wording make sense to you?

**Review by keyringcore — APPROVED** [review]

**Review by rob-Hitchens — APPROVED** [review]

**Pull request merged**